From 069343ef2025b751397cc74952875c12f6940c9d Mon Sep 17 00:00:00 2001
From: Vitaly Dyachkov <vitaly@igalia.com>
Date: Wed, 13 Mar 2024 05:50:19 -0700
Subject: [PATCH] [WPE][GTK] Unreviewed, fix build for Ubuntu LTS after
 276012@main

* Source/WebKit/PlatformGTK.cmake:
* Source/WebKit/PlatformWPE.cmake:

Canonical link: https://commits.webkit.org/276020@main
---
 Source/WebKit/PlatformGTK.cmake | 2 +-
 Source/WebKit/PlatformWPE.cmake | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

Upstream-Status: Backport [https://commits.webkit.org/276020@main]

diff --git a/Source/WebKit/PlatformWPE.cmake b/Source/WebKit/PlatformWPE.cmake
index 92825501798b..be960eda81a4 100644
--- a/Source/WebKit/PlatformWPE.cmake
+++ b/Source/WebKit/PlatformWPE.cmake
@@ -276,7 +276,7 @@ list(APPEND WebKit_DEPENDENCIES
     webkitwpe-forwarding-headers
 )
 
-if (${GI_VERSION} VERSION_GREATER_EQUAL 1.79.2)
+if (GI_VERSION VERSION_GREATER_EQUAL 1.79.2)
     set(USE_GI_FINISH_FUNC_ANNOTATION 1)
 else ()
     set(USE_GI_FINISH_FUNC_ANNOTATION 0)
-- 
2.34.1

